third_party/terraform/website/docs/guides/version_3_upgrade.html.markdown

```diff
@@ -238,6 +239,14 @@ introducing an incompatibility for users who specify config in JSON instead of
 HCL or who use `dynamic` blocks. See more details in the [Attributes as Blocks](https://www.terraform.io/docs/configuration/attr-as-blocks.html)
 documentation.
 
+## Resource: `google_project_service`
+
+### `bigquery-json.googleapis.com` service can no longer be specified
+
+`bigquery-json.googleapis.com` is being renamed to `bigquery.googleapis.com` in
+the upstream API. As a result, `bigquery-json.googleapis.com` has been
+disallowed.
```

Comment:
Maybe add "Please use `bigquery.googleapis.com` - the provider will automatically convert between them as the upstream API migration continues."